Makefile

```diff
@@ -31,11 +31,12 @@ DOCS_BUILD_TARGET ?= /docs/source/getting-started/master.adoc
 
 GOOS ?= $(shell go env GOOS)
 GOARCH ?= $(shell go env GOARCH)
+HOST_BUILD_DIR=$(BUILD_DIR)/$(GOOS)-$(GOARCH)
```

Comment:
@cfergeau Any specific reason not directly using the `$(BUILD_DIR)` one instead creating a new directory?

Reply:
The `out` directory contains `out/macos-amd64/crc`, `out/linux-amd64/crc` and `out/win-amd64/crc.exe`, it felt odd to add a `out/crc-embedder` binary rather than putting it in the correct `out/$os-$arch/` directory.

Reply:
Ack 👍

Reply:
actually, I suspect on macos, `$(GOOS)-$(GOARCH)` is going to be `darwin-amd64`, not `macos-amd64`, so an additional commit may be needed.

Reply:
Confirmed, but this does not cause a build failure, it just created `out/darwin-amd64/crc-embedder` in addition to `out/macos-amd64/crc`. I'll keep it that way for now.